<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mywindow</class>
 <widget class="QMainWindow" name="mywindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>mywindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="select_page">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 0);
background-color: rgb(0, 0, 0);</string>
     </property>
     <widget class="QPushButton" name="login">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>111</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(111, 111, 111);</string>
      </property>
      <property name="text">
       <string>LOGIN</string>
      </property>
     </widget>
     <widget class="QPushButton" name="signup">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>140</y>
        <width>141</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(111, 111, 111);</string>
      </property>
      <property name="text">
       <string>SIGN UP</string>
      </property>
     </widget>
     <widget class="QPushButton" name="about">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>478</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Script</family>
       </font>
      </property>
      <property name="text">
       <string>About</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>341</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 0);
font: 18pt &quot;Segoe Script&quot;;</string>
      </property>
      <property name="text">
       <string>BACHELOR'S AUDIT</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="login_page">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QPushButton" name="login_pushbutton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>210</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>8514oem</family>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(94, 255, 166);
background-color: rgb(0, 85, 127);</string>
      </property>
      <property name="text">
       <string>LOGIN</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="login_email">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>110</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="placeholderText">
       <string>E-mail</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="login_password">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Snap ITC</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>E-mail</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Snap ITC</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="signup_page">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLineEdit" name="signup_name">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>431</width>
        <height>28</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="signup_mobile">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>110</y>
        <width>241</width>
        <height>28</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Mobile No.</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="signup_address">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>150</y>
        <width>431</width>
        <height>28</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Address</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="signup_email">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>190</y>
        <width>431</width>
        <height>28</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>E-mail</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="signup_password">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>230</y>
        <width>241</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Wide Latin</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Wide Latin</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Mobile No.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Wide Latin</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>190</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Wide Latin</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>E-mail</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>230</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Wide Latin</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>290</y>
        <width>83</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>8514oem</family>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 85, 127);</string>
      </property>
      <property name="text">
       <string>SIGN UP</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="fourth_page">
     <widget class="QPushButton" name="exit_button">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>520</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>0</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Please enter mess cost details...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>1. House Rent.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>150</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>2. Gass Bill.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>190</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>3. Current Bill</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>230</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>4. Wifi Bill</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>270</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>5. Others Cost</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="gass_bill">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>150</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="house_rent">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="current_bill">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>190</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="wifi_bill">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>230</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="other_cost">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>270</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="next_buton_1">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>330</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>NEXT</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>381</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Enter your Meal &amp; Bazar Expenses.....</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>1. Total Mess Member.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>2. Total Bazar Cost.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>200</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>3. Total Meal.</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="total_mess_member">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="total_bazar_cost">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>160</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="total_meal">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>200</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="next_bytton_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>250</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>NEXT</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>160</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Meal Rate</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>40</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Mass Month Details</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string> Total meal</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>210</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string> Bazar Cost</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>260</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Other's Cost</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>360</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Per Person Total Cost</string>
      </property>
     </widget>
     <widget class="QLabel" name="labeltotalmeal">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>120</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelmealrate">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>170</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelbazar">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>220</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelother">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>260</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelperperson">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>370</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>310</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Total cost</string>
      </property>
     </widget>
     <widget class="QLabel" name="labeltotalcost">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>310</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>490</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>731</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 10pt &quot;Sitka&quot;;
</string>
      </property>
      <property name="text">
       <string>Alhamdulillah; By using this application you can get very few benefits.Notable</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>731</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 10pt &quot;Sitka&quot;;</string>
      </property>
      <property name="text">
       <string> among them, the details of the life of a bachelor! For example, you will get a</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>149</y>
        <width>731</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 10pt &quot;Sitka&quot;;</string>
      </property>
      <property name="text">
       <string> sample of all expenses for the whole month and at the end of the month how</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>741</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 10pt &quot;Sitka&quot;;</string>
      </property>
      <property name="text">
       <string> much money was spent per person.I hope, I can benefit you a little in this</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>260</y>
        <width>731</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 10pt &quot;Sitka&quot;;</string>
      </property>
      <property name="text">
       <string>small work, in sha Allah!</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>320</y>
        <width>331</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 10pt &quot;Sitka&quot;;
color: rgb(85, 170, 0);</string>
      </property>
      <property name="text">
       <string>E-mail :mizanulhoque2021@gmail.com</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>360</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Sitka&quot;;
color: rgb(85, 170, 0);
</string>
      </property>
      <property name="text">
       <string>Mobile No : 01321872868</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>400</y>
        <width>671</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 10pt &quot;Sitka&quot;;</string>
      </property>
      <property name="text">
       <string>If You Face Any Problem Regarding Bachelor Audit App or want to know </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>430</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 10pt &quot;Sitka&quot;;</string>
      </property>
      <property name="text">
       <string>anything please contact us.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>500</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 10pt &quot;Sitka&quot;;
color: rgb(255, 170, 255);</string>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
